### PR TITLE
Update FD.yaml

### PR DESCRIPTION
**Pull request by Karthi8883**
Points are required to monitor supply and return dampers status

### CODE REVIEW

**Comment by tasodorff** [comment]
validator errors

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 1]
```diff
@@ -161,6 +161,21 @@ literals:
 - supply_air_temperature_setpoint
 - supply_fan_current_sensor
 - supply_fan_power_sensor
+- supply_air_damper_open_status:
```
Comment (cstirdivant) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:164]: is it possible to model this as one field `supply_air_damper_status` ? or are there legitimately two separate fields for open and closed?

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/fields/telemetry_fields.yaml (hunk 1 above)
Comment (Karthi8883) [reply ontology/yaml/resources/fields/telemetry_fields.yaml:164]: There are two fields. The status in Open field will be active only when the damper is fully open and the status in the Closed field will be active only when the damper is fully closed. If the damper is for example is stuck in an intermediate position, neither of the fields will be active.

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 2]
```diff
@@ -161,6 +161,21 @@ literals:
...
+- supply_air_damper_closed_status:
+  - OPEN
+  - CLOSED
+- return_air_damper_open_status:
+  - OPEN
+  - CLOSED
+- return_air_damper_closed_status:
+  - OPEN
+  - CLOSED
+- high_temperature_alarm:
```
[the remark this answers is not in the document]
Comment (Karthi8883) [reply ontology/yaml/resources/fields/telemetry_fields.yaml:176]: There is a sensor placed in the zone where the panel is located, this field is active when the sensor reads a high value. Can we model it as zone_high_temperature_alarm?

[the remark this answers is not in the document]
Comment (tasodorff) [reply ontology/yaml/resources/fields/telemetry_fields.yaml:176]: should be `high_zone_air_temperature_alarm` in that case. there are other high temp alarms that follow that precedent.

File: ontology/yaml/resources/SAFETY/entity_types/FD.yaml [hunk 3]
```diff
@@ -19,4 +19,11 @@ FD_STD:
   implements:
   - FD
   uses:
-  - damper_status
\ No newline at end of file
+  - damper_status
+  opt_uses:
+  - supply_air_damper_open_status
```
Comment (tasodorff) [inline ontology/yaml/resources/SAFETY/entity_types/FD.yaml:24]: if there is a supply air damper on the same device, then what is the damper above? either remove these from this FD or differentiate the field above.

**Review by tasodorff — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/SAFETY/entity_types/FD.yaml (hunk 3 above)
Comment (Karthi8883) [reply ontology/yaml/resources/SAFETY/entity_types/FD.yaml:24]: This field was already existing. Did not want to break the existing field, hence modeled it in opt_uses with both supply and return dampers.

File: ontology/yaml/resources/fields/telemetry_fields.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (Karthi8883) [reply ontology/yaml/resources/fields/telemetry_fields.yaml:176]: Updated

File: ontology/yaml/resources/SAFETY/entity_types/FD.yaml (hunk 3 above)
Comment (Karthi8883) [reply ontology/yaml/resources/SAFETY/entity_types/FD.yaml:24]: The field was already existing. Did not want to break existing model, hence modeled under opt_uses for both supply and return dampers

Comment (tasodorff) [reply ontology/yaml/resources/SAFETY/entity_types/FD.yaml:24]: but the supply and return on what? typically a fire smoke damper is its own stand-alone damper, and there is no reference to supply or return air.

Comment (tasodorff) [reply ontology/yaml/resources/SAFETY/entity_types/FD.yaml:24]: i think you are modeling too many dampers as part of the FD.

Comment (Karthi8883) [reply ontology/yaml/resources/SAFETY/entity_types/FD.yaml:24]: Noted, we will close this pull request and model a generic FD with two dampers as a separate pull request.

**Pull request closed without merging**